spec/system/planning_applications/index_spec.rb

```diff
@@ -69,6 +71,28 @@
       expect(page).to have_current_path(/sign_in/)
       expect(page).to have_content("You need to sign in or sign up before continuing.")
     end
+
+    scenario "On login, assessor gets redirected to a view with its own and unassigned Planning Applications" do
+      visit planning_applications_path(assessor: "assessor")
+
+      within("#in_assessment") do
+        expect(page).to have_link(planning_application_1.reference)
+        expect(page).to have_link(planning_application_2.reference)
+        expect(page).not_to have_link(planning_application_3.reference)
+      end
+    end
+
```

Comment:
given application_3 belongs to assessor_2 that's the reason it shouldn't be displayed here - perhaps its best to create assessor 2 and application3 inside the block  instead of using "let!" at the top, so the test is more explicit?